core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.component.ts

```diff
@@ -192,7 +192,10 @@ angular.module('oppia').component('topicsAndSkillsDashboardPage', {
       };
 
       ctrl.createSkill = function() {
-        SkillCreationService.createNewSkill();
+        let allSkillNames = ctrl.untriagedSkillSummaries.map(
+          skillSummary => (skillSummary.description || '').toUpperCase()
+        );
+        SkillCreationService.createNewSkill([], allSkillNames);
```

Comment:
Hm, would it make sense to store (and keep updated) the skill names in the SkillCreationService instead of passing it along?